srcDLL/DllMain.cpp

```diff
@@ -143,12 +141,13 @@ void LocateVolFiles(const std::string& relativeDirectory)
 			const auto extension = ToLower(filePath.extension().string());
 
 			if (extension == ".vol") {
```

Comment:
We could generalize this method by making the desired extension a parameter.

Reply:
I suspect this branch is going to balloon in scope...

Reply:
I kind of do that don't I. Perhaps I should have pushed some of those thoughts into an issue.